README.md

```diff
@@ -15,6 +15,11 @@ Documentation
 
 Documentation is available on [**docs.sylius.com**](https://docs.sylius.com/en/latest/api/index.html).
 
+AdminApiBundle
+--------------
+
+AdminApiBundle [Documentation](docs/index.md)
```

Comment:
I believe we should just change the previous section link :)

Reply:
:)